website/versioned_docs/version-0.62/rnw-dependencies.md

```diff
@@ -0,0 +1,50 @@
+---
+id: version-0.62-rnw-dependencies
+title: System Requirements
+original_id: rnw-dependencies
+---
+
+
+# Check and install dependencies
```

Comment:
Headings are off. This should be a level 2 header to match the rest of the doc? 

Also, the sections are a bit unclear. Here is my suggested revision of this page:

## Windows Development Dependencies
You can run React-Native for Windows apps only on Windows 10 devices with Windows version: 10.0.16299.0 (aka 1709, aka Redstone 3, aka Fall Creators Update) or higher. Some features may not work on all versions. See [Windows 10 Compatibility](win10-compat.md) for version support details.

To develop React-Native for Windows apps, you need to install several dependencies.

### Script to check and install all dependencies
To check or install dependencies, run the script [rnw-dependencies.ps1](https://github.com/microsoft/react-native-windows/blob/master/vnext/Scripts/rnw-dependencies.ps1) in an elevated PowerShell window.

**Run this command:** 
`Start-Process -Verb RunAs powershell -ArgumentList @("-command", "iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/microsoft/react-native-windows/master/vnext/Scripts/rnw-dependencies.ps1'))")`

### Manual setup
Alternatively, you can setup your environment manually:
- Ensure Developer Mode is turned ON in Windows Settings App.
- Install [Visual Studio 2019](https://www.visualstudio.com/downloads) **with the following options checked**:
  - Workloads
    - Universal Windows Platform development
    - Desktop development with C++
  - Individual Components
    - Development activities
      - Node.js development support (optional)

... (the rest of the doc can stay as is).

Reply:
thanks for writing this up, I added it.